models/incident.js

```diff
@@ -17,6 +17,7 @@ var schema = new mongoose.Schema({
   updatedAt: Date,
   storedAt: Date,
   assignedTo: String,
+  assignedToId: String,
```

Comment:
I believe this should be refactored into a full association. `assignedTo` should be removed and replaced with _assignedTo, and should use an ObjectId. Talk to Aman about this if you're not sure how.